spring-test/spring-test.gradle

```diff
@@ -50,6 +50,8 @@ dependencies {
 	optional("io.projectreactor:reactor-test")
 	optional("org.jetbrains.kotlinx:kotlinx-coroutines-core")
 	optional("org.jetbrains.kotlinx:kotlinx-coroutines-reactor")
+	optional('io.micrometer:micrometer-observation')
+	optional('io.micrometer:context-propagation')
```

Comment:
Please sort dependencies alphabetically in the build script.

Reply:
I've changed the order